src/xenia/base/string_util.h

```diff
@@ -24,13 +24,13 @@ namespace string_util {
 
 inline std::string to_hex_string(uint32_t value) {
   char buffer[21];
-  std::snprintf(buffer, sizeof(buffer), "%08" PRIX32, value);
+  snprintf(buffer, sizeof(buffer), "%08" PRIX32, value);
   return std::string(buffer);
 }
 
 inline std::string to_hex_string(uint64_t value) {
   char buffer[21];
-  std::snprintf(buffer, sizeof(buffer), "%016" PRIX64, value);
+  snprintf(buffer, sizeof(buffer), "%016" PRIX64, value);
```

Comment:
I don't think this belongs to this PR. Why did you drop `std::`?

Reply:
Clarified the drop of std:: in the commit message, copying here:
This seems to be an issue with Microsoft declaring a snprintf macro in an included header which causes issues.
Related issues:

nlohmann/json#1408
kodi-pvr/pvr.vuplus#162